plugins/soccer-scoreboard/base_odds_manager.py

```
@@ -112,8 +112,14 @@ def get_odds(
         cached_data = self.cache_manager.get(cache_key)
 
         if cached_data:
-            self.logger.info(f"Using cached odds from ESPN for {cache_key}")
-            return cached_data
+            # Filter out the "no_odds" marker – it should not be returned
```

Ruff (0.15.0)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)